rules/remote-desktop-manager/rule.md

```diff
@@ -34,4 +34,4 @@ To install the application, check https://devolutions.net/remote-desktop-manager
 
```

Comment:
I think to keeping the links in a markdown format is a bit nicer here. (I know this isn't part of your change) 

From:
To install the application, check https://devolutions.net/remote-desktop-manager/home/downloadfree 

To:
To install the application, check [devolutions.net](https://devolutions.net/remote-desktop-manager/home/downloadfree )